### PR TITLE
bug 1641059 added --rolebinding-name option

### PR DESCRIPTION
**Pull request by sheriff-rh**
Bug:
https://bugzilla.redhat.com/show_bug.cgi?id=1641059

Added an item in table 2 to reflect the usage of the `--rolebinding-name=` option. 

@openshift/team-documentation PTAL
@stuartchuan PTAL

### CODE REVIEW

**Comment by ahardin-rh** [comment]
LGTM :fireworks:

File: admin_guide/manage_rbac.adoc [hunk 1]
```diff
@@ -631,6 +631,9 @@ cluster role bindings use non-namespaced resources.
 |`$ oc adm policy remove-cluster-role-from-group _<role>_ _<groupname>_`
 |Removes a given role from specified groups for all projects in the cluster.
 
+|`--rolebinding-name=`
+|Can be used with `oc adm policy` commands to retain the rolebinding name assigned to a user.
+
```
Comment (stuartchuan) [inline admin_guide/manage_rbac.adoc]: The option `--rolebinding-name=` could be apply for not only user, but also for group.
And it should be added to table 1 also.

Comment (sheriff-rh) [reply admin_guide/manage_rbac.adoc]: Thanks @stuartchuan , I added `assigned to a user or group` and added it to table 1 as well.

**Comment by sheriff-rh** [comment]
Commited and pushed QA's recommendations; merging.

**Pull request merged**